certora/helpers/DummyERC20Impl.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+// with mint
+contract DummyERC20Impl {
```

Comment:
I'm missing the point of this contract. We have an ERC20 impl, why do we need that ?

Reply:
This was created before as a basic implementation. We use this on most projects but are working on having a more diverse set up ERC20 implementations. i can replace it with OZ implementation if you'd like.